copier/vcs.py

```diff
@@ -80,7 +80,7 @@ def get_repo(url: str) -> OptStr:
     if url.startswith("git+"):
         url = url[4:]
     elif url.startswith("https://") and not url.endswith(GIT_POSTFIX):
-        url = "".join((url, GIT_POSTFIX))
+        url = "".join((url, ".git"))
```

Comment:
Oh please leave it as it was and fix the variable declaration above. I didn't notice it's a tuple. It should be a str.

Thanks and sorry for that.